suite-native/assets/src/components/DiscoveryAssetsLoader.tsx

```diff
@@ -1,27 +1,27 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
 
 import { ListItemSkeleton, HStack, Text } from '@suite-native/atoms';
 import { Icon } from '@suite-common/icons';
 import { useTranslate } from '@suite-native/intl';
-import { selectIsAccountsListEmpty } from '@suite-common/wallet-core';
 
-const EMPTY_LIST_SKELETON_COUNT = 3;
-const NONEMPTY_LIST_SKELETON_COUNT = 1;
+const DEFAULT_LIST_SKELETON_COUNT = 3;
+const MINIMUM_LIST_SKELETON_COUNT = 1;
 
-export const DiscoveryAssetsLoader = () => {
+export const DiscoveryAssetsLoader = ({ numberOfAssets }: { numberOfAssets: number }) => {
     const { translate } = useTranslate();
-    const isListEmpty = useSelector(selectIsAccountsListEmpty);
+    const isListEmpty = numberOfAssets === 0;
```

Comment:
Please use `selectIsPortfolioEmpty` so it's consistent. Any reason why not?

Reply:
It returns false if discovery is in progress which is not desired here

Reply:
And why not `selectIsAccountsListEmpty`? Same thing without the discovery check

Reply:
I need `numberOfAssets` anyway for other calculations so I don't see point introduction another selector to this component if can do simple check like this